src/components/Feed/Feed.css

```diff
@@ -0,0 +1,17 @@
+main {
+    animation: soft-rainbow 20s infinite; 
+}
+
+@keyframes soft-rainbow {
+  0%{box-shadow: 6px 12px 4px #fefeff;}	
```

Comment:
Killing it with the animations - let's potentially shave this down at some point